### PR TITLE
CSUB-1048 Payouts not displaying

### PR DESCRIPTION
**Pull request by frankli-dev**

### CODE REVIEW

**Review by juanma1996 — APPROVED** [review]

File: src/config/networks.ts [hunk 1]
```diff
@@ -95,8 +95,8 @@ const makeNetworkList = () => {
         },
       },
       namespace: '09573a3526818a8ecd6eb92f60f1175d',
-      subscanEndpoint: 'http://127.0.0.1:4399',
-      subscanUrl: 'https://creditcoin3-testnet.subscan.io/',
+      subscanEndpoint: 'https://creditcoin3-dev.api.subscan.io',
+      subscanUrl: 'https://subscan-cc3-devnet.creditcoin.network',
```
Comment (juanma1996) [inline src/config/networks.ts:99]: This is the block explorer url, right?
In that case it should be: https://creditcoin3-dev.subscan.io

File: src/config/networks.ts [hunk 2]
```diff
@@ -109,8 +109,8 @@ const makeNetworkList = () => {
       },
     },
     namespace: '09573a3526818a8ecd6eb92f60f1175d',
-    subscanEndpoint: 'http://127.0.0.1:4399',
-    subscanUrl: 'https://creditcoin3-testnet.subscan.io/',
+    subscanEndpoint: 'https://creditcoin3-testnet.api.subscan.io',
+    subscanUrl: 'https://subscan-cc3-testnet.creditcoin.network',
```
Comment (juanma1996) [inline src/config/networks.ts:113]: This is the block explorer url, right?
In that case it should be: https://creditcoin3-testnet.subscan.io

File: src/config/networks.ts [hunk 3]
```diff
@@ -123,7 +123,7 @@ const makeNetworkList = () => {
     },
     namespace: 'creditcoin-mainnet',
     subscanEndpoint: 'http://127.0.0.1:4399',
-    subscanUrl: 'https://creditcoin3.subscan.io/',
+    subscanUrl: 'https://creditcoin3.subscan.io',
```
Comment (juanma1996) [inline src/config/networks.ts:126]: This need to matches whatever it's on the other networks, I think this one is fine. But just to remember double check it.

File: src/config/networks.ts [hunk 4]
```diff
@@ -136,7 +136,7 @@ const makeNetworkList = () => {
         },
       },
       subscanEndpoint: 'http://127.0.0.1:4399',
-      subscanUrl: 'https://creditcoin3-testnet.subscan.io/',
+      subscanUrl: 'https://creditcoin3-testnet.subscan.io',
```
Comment (juanma1996) [inline src/config/networks.ts:139]: This need to matches whatever it's on the other networks, I think this one is fine. But just to remember double check it.

**Review by juanma1996 — APPROVED** [review]

**Review by juanma1996 — APPROVED** [review]

**Review by pLabarta — APPROVED** [review]

**Pull request merged**